Pop-Grupa-A-Backend/migrations/versions/b1056134dfb4_computation_task_columns_modification.py

```diff
@@ -0,0 +1,42 @@
+"""computation task columns modification
+
+Revision ID: b1056134dfb4
+Revises: a615c14b885c
+Create Date: 2019-10-21 18:20:42.626007
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b1056134dfb4'
+down_revision = 'a615c14b885c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('ComputationAccount', 'password')
```

Comment:
Dlaczego dropujecie password z użytkownika?

Reply:
to też nie mój kod